downgrade

```diff
@@ -209,6 +233,7 @@ main() {
     # shellcheck disable=SC2207
     installed=($(previously_installed "$term"))
     current=$(currently_installed "$term")
+    [[ -z "$current" ]] && gettext "No package with name $term found"
     process_term "$term" || exit_code=1
```

Comment:
This isn't _exactly_ right. We now fail if we're the term isn't currently installed, but there's actually no reason to fail there. If I want to install some older `firefox`, even if I don't currently have `firefox` installed, I think that should work -- and it used to.

I think you want:

```suggestion
    if ! process_term "$term"; then
      gettext "Unable to downgrade $term"
      exit_code=1
    fi
```

Reply:
Yes I overlooked that possibility. Your suggestion will keep that functionality going, committed.